website/www/site/content/en/community/mascot.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Beam Mascot"
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Beam Mascot Design
+
+This page contains Apache Beam's mascot designs. 
+
+Beam firefly is cute, friendly, agile, easy to use, and its main objective is to fetch streams and batches of data and process it. The mascot’s model sheet is useful to understand its features, capabilities, as well as its morphology.  The original design of the mascot was created by [Julian G. Bruno](https://www.artstation.com/jbruno) and was donated to the Apache Beam community under the [Apache license 2.0](https://www.apache.org/licenses/LICENSE-2.0). 
+
+You can browse the original mascot and its adaptations in different sizes and image formats in [this directory](https://github.com/apache/beam/tree/mascot-upload/website/www/site/static/images/mascot). 
```

Comment:
The directory is aprt of the github repository. it is not yet merged, but once it is, it will work. I've fixed the apache licenses. Can you try taking a lo0ok? Than ks!